### ISSUE TITLE
Unable to save Highrise CRM feed on multisite

### ISSUE THREAD

**Issue by prestodesigns** [bhays/gravity-forms-highrise-crm#8]
The Highrise Feeds will save on the primary site, but on the secondary site it allows me to create the feed and save it.  It also give the confirmation popup, but when going back to the feed list there is still no feed created.

**Comment by bhays** [comment]
Feeds are saving fine for me on secondary sites. 

When you go back to the feed list are you clicking the back button? That may be causing the newly created entry to not show.

Alternatively, have you looked at your database to ensure the tables were created correctly? There should be tables named `wp_rg_highrise` and `wp_2_rg_highrise` or similar depending on your database prefix.

**Comment by prestodesigns** [comment]
No, I am using the "back to list" link.  I also checked and both of 
those tables are in my database.  Any other ideas on what to check?

**Comment by bhays** [comment]
Are there any rows of data in the databases?
